### PR TITLE
Foo/coaching progress Coaching Progress Dashboard

### PR DESCRIPTION
**Pull request by cwbmuller**
I improved the filtering on the filtering on the assessment review dashboard

screenshot 2019-01-25 at 11 28 38

After filtering you get helpful additional options & I also tweaked the columns to be more helpful

screenshot 2019-01-25 at 11 28 53

I created another dashboard for course progress and linked the two together

screenshot 2019-01-25 at 11 30 23

Once a course is selected it shows user course progress. The Coach can then drill down into a user to see module progress.

screenshot 2019-01-25 at 11 30 37

Module progress then shows if a module/unit is complete

screenshot 2019-01-25 at 11 31 03

Finally, I added a link to dashboard in menu if the user is a coach

screenshot 2019-01-25 at 11 35 41

### CODE REVIEW

File: wp-content/themes/fundawande/templates/lms/template-coach-dash.twig [hunk 1]
```diff
@@ -71,9 +86,9 @@
                                 <th>Assessment</th>
                                 <th>Module</th>
                                 <th>Submission date</th>
-                                <th>Feedback</th>
+                                <th>Feedback?</th>
```
Comment (JasonTame) [inline wp-content/themes/fundawande/templates/lms/template-coach-dash.twig]: This label and the options within the dropdown might need re-wording.

Comment (JasonTame) [inline wp-content/themes/fundawande/templates/lms/template-coach-dash.twig]: This label and the options within the dropdown might need re-wording.

File: wp-content/themes/fundawande/templates/lms/template-coach-dash.twig [hunk 2]
```diff
@@ -11,57 +11,72 @@
...
+
+                            <div class="col-auto px-3 mb-4 mb-md-0">
+                                <select type="select" name="fw_course" class="form-control custom-select coach-filter"   id="courseSelect"  style="max-width:150px" placeholder="" value="" required>
+                                    <option value="" disabled  selected >Choose a course</option>
+                                    {% for course in courses %}
+                                        <option value="{{course.ID }}" {% if selected_course ==  course.ID %}selected{% endif %}  >{{course.post_title}}</option>
+                                    {% endfor %}
+                                </select>
+                            </div>
+                            <div class="col-auto px-3 ">
```
Comment (JasonTame) [inline wp-content/themes/fundawande/templates/lms/template-coach-dash.twig]: If I understand the function of this filter correctly, it filters learners based on who their assigned coach is? Perhaps its a filter that should only be visible if the user accessing the coach dashboard is an administrator?

File: wp-content/themes/fundawande/templates/lms/template-coach-dash.twig [hunk 3]
```diff
@@ -11,57 +11,72 @@
...
+                                <select type="select" name="coach" class="form-control custom-select coach-filter " id="coachSelect" style="max-width:150px"  placeholder="" value="" >
+                                    <option value="" disabled selected >Filter coaches</option>
```
Comment (JasonTame) [inline wp-content/themes/fundawande/templates/lms/template-coach-dash.twig]: If I understand the function of this filter correctly, it filters learners based on who their assigned coach is? Perhaps its a filter that should only be visible if the user accessing the coach dashboard is an administrator?

File: wp-content/themes/fundawande/templates/lms/template-coach-dash.twig (hunk 1 above)
Comment (cwbmuller) [reply wp-content/themes/fundawande/templates/lms/template-coach-dash.twig]: Yeah, let's see what they say

File: wp-content/themes/fundawande/templates/lms/template-coach-dash.twig (hunk 3 above)
Comment (cwbmuller) [reply wp-content/themes/fundawande/templates/lms/template-coach-dash.twig]: Ya, i'm just worried about limiting it for now. So going to leave as all coach filter for now

**Pull request merged**